src/App.js

```diff
@@ -44,6 +45,7 @@ const App = () => {
           duration: 5000,
         }}
       />
+      <Suspense fallback={<Loader />}>
```

Comment:
Also, Since we are now loading all routes in the App.js files. Perhaps we could remove the suspense component from [index.js](https://github.com/smswithoutborders/smswithoutborders.com/blob/dev/src/pages/index.js) as it would be doing basically the same thing already handled here

Reply:
ok.will make this corrections

Reply:
@nSedrickm  using my App.js js component in index.js without the suspense component yields the following error
`Uncaught Error: App suspended while rendering, but no fallback UI was specified.
Add a <Suspense fallback=...> component higher in the tree to provide a loading indicator or placeholder to display.`
so I have left the suspense component with the default Splashscreen fallback

Reply:
Thanks @teyim. On second look, the suspense component in App.js helps when loading translation files with reacti18next so great we are leaving it in place